Actions/AL-Go-Helper.ps1

```diff
@@ -2026,3 +2027,35 @@ function Determine-ArtifactUrl {
     }
     return $artifactUrl
 }
+
+function Retry-Command {
+    param(
+        [Parameter(Mandatory = $true)]
+        [ScriptBlock]$Command,
+        [Parameter(Mandatory = $true)]
+        [int]$MaxRetries,
+        [Parameter(Mandatory = $false)]
+        [int]$RetryDelaySeconds = 5
+    )
+
+    $retryCount = 0
+    while ($retryCount -lt $MaxRetries) {
+        try {
+            & $Command
```

Comment:
I typically use Invoke-command to invoke scriptblocks - are you sure that & is the right mechanism? (I thought that was for commands only, but looking at the code, it looke like it works)

Reply:
I think both works. Taking your suggestion and using Invoke-Command